geom/encoding/geojson/geojson.go

```diff
@@ -0,0 +1,122 @@
+package geojson
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/terranodo/tegola/geom"
+)
+
+type GeoJSONType string
+
+const (
+	PointType              GeoJSONType = "Point"
+	MultiPointType         GeoJSONType = "MultiPoint"
+	LineStringType         GeoJSONType = "LineString"
+	MultiLineStringType    GeoJSONType = "MultiLineString"
+	PolygonType            GeoJSONType = "Polygon"
+	MultiPolygonType       GeoJSONType = "MultiPolygon"
+	GeometryCollectionType GeoJSONType = "GeometryCollection"
+)
+
+type ErrUnknownGeometry struct {
+	Geom geom.Geometry
+}
+
+func (e ErrUnknownGeometry) Error() string {
+	return fmt.Sprintf("unknown geometry: %T", e.Geom)
+}
+
+type Geometry struct {
+	geom.Geometry
+}
+
+func (geo Geometry) MarshalJSON() ([]byte, error) {
+	type coordinates struct {
+		Type  GeoJSONType `json:"type"`
+		Coord interface{} `json:"coordinates,omitempty"`
```

Comment:
`Coords` to be consistent with `collection.Geometries`

Reply:
Just to clarify change `Coord` to `Coordinates`. If so 👍